core/src/main/java/org/frankframework/javascript/FrankJ2V8LibraryLoader.java

```diff
@@ -162,41 +153,23 @@ static boolean extract(final String fileName, final String mappedName, final Str
 		return false;
 	}
 
-	static void chmod(final String permision, final String path) {
-		if (isWindows()) {
-			return;
-		}
-		try {
-			Runtime.getRuntime().exec(new String[] { "chmod", permision, path }).waitFor(); //$NON-NLS-1$
-		} catch (Throwable e) {
-		}
-	}
-
-	static String getOsName() {
-		return System.getProperty("os.name") + System.getProperty("java.specification.vendor");
-	}
-
-	static boolean isWindows() {
-		return getOsName().startsWith("Windows");
-	}
-
-	static boolean isMac() {
-		return getOsName().startsWith("Mac");
+	private static boolean isWindows() {
+		return OS_NAME.startsWith("Windows");
 	}
 
-	static boolean isLinux() {
-		return getOsName().startsWith("Linux");
+	private static boolean isMac() {
+		return OS_NAME.startsWith("Mac");
 	}
 
-	static boolean isNativeClient() {
-		return getOsName().startsWith("nacl");
+	private static boolean isLinux() {
+		return OS_NAME.startsWith("Linux");
 	}
 
-	static boolean isAndroid() {
-		return getOsName().contains("Android");
```

Comment:
`isAndroid()`? 🤔 
Die had ik niet verwacht!

Reply:
Tja, de file is ooit overgekopieerd van eclipsesource/j2v8 :)